Multisig/UI/Safe Management/Create Safe/CreateSafeFlow.swift

```diff
@@ -108,21 +108,27 @@ class CreateSafeFlow: UIFlow, ASAuthorizationControllerPresentationContextProvid
     }
 
     func googleLogin() {
-        let loginModel = GoogleWeb3AuthLoginModel(authorizationComplete: {
-            let view = self.factory.safeCreatingViewController()
-            view.onSuccess = {
-                self.stop(success: true)
+        let authorizationComplete = { [weak self] in
+            let view = self?.factory.safeCreatingViewController()
+            view?.onSuccess = {
+                self?.stop(success: true)
             }
-            self.show(view)
-
-        } , keyGenerationComplete: { [weak self] key, email in
+            self?.show(view!)
```

Comment:
the standard handling of the `[weak self]` when there's more than one call using it is to add guard in the closure, like this:

```
 let authorizationComplete = { [weak self] in
            guard let self = self else { return }
            let view = self.factory.safeCreatingViewController()
            view.onSuccess = { [weak self] in
                self.stop(success: true)
            }
            self.show(view)
```